pyserini/prebuilt_index_info.py

```diff
@@ -151,5 +151,65 @@
         "documents": 4170312,
         "unique_terms": -1,
         "downloaded": False
+    },
+    "trec-covid5": {
+        "description": "Trec Covid Round 5",
+        "urls": [
+            "https://www.dropbox.com/s/9hfowxi7zenuaay/lucene-index-cord19-abstract-2020-07-16.tar.gz?dl=1"
+        ],
+        "md5": "c883571ccc78b4c2ce05b41eb07f5405",
+        "size compressed (bytes)": 2796524,
+        "total_terms": -1,
+        "documents": 18,
```

Comment:
these stats don't seem right?

Reply:
This method should check out:
https://github.com/castorini/pyserini/blob/master/pyserini/index/_base.py#L180

Reply:
Sorry, the number should be fixed now.